auslib/db.py

```diff
@@ -1020,6 +1020,7 @@ def __init__(self, db, metadata, dialect):
                            Column('headerArchitecture', String(10)),
                            Column('comment', String(500)),
                            Column('whitelist', String(100)),
```

Comment:
Whitelist also maps to Releases, so you should add a foreign key for it too. Please add a test as well.